packages/ui/src/utils/get-required-properties-schema.test.ts

```diff
@@ -4,6 +4,20 @@ import { KaotoSchemaDefinition } from '../models/kaoto-schema';
 describe('getRequiredPropertiesSchema()', () => {
   const schema = {
     type: 'object',
+    definitions: {
```

Comment:
Could we reuse the previous file from the `stubs` folder, and perhaps just manually add this `$ref` property?

Reply:
We have a custom made schema  in this case and for `getUserUpdatedProperties()` test so we cant re-use the `restSchema` stub here. Hence created a a different stub for this!